pyproject.toml

```diff
@@ -15,6 +15,12 @@ dependencies = [
     "distro>=1.7.0, <2",
     "sniffio",
     "cached-property; python_version < '3.8'",
```

Comment:
Can’t we just force python 3.10+

Reply:
I dont think we force this on client side currently.